cdec/installation-manager-resources/src/main/resources/im-scripts/install-codenvy.sh

```diff
@@ -1524,8 +1561,8 @@ doCheckAvailableResourcesOnNodes() {
                 ;;
         esac
 
-        # check on OS CentOS 7
-        if [[ "${osType}" != "CentOS" || "7.1" > "${osVersion}" ]]; then
+        # check on CentOS or RHEL OS of version >= 7.1
+        if [[ "${osType}" != "CentOS" || "${osVersion}" < "${MIN_OS_VERSION}" ]]; then
```

Comment:
as discussed, checking /etc/redhat-release should imply that osType is named RHEL, not centos

then the check osType != "RHEL" is meaningful for bot RHEL (and derivatives like Centos, etc)